internal/server/files.go

```diff
@@ -0,0 +1,184 @@
+package server
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/labstack/echo/v4"
+)
+
+type files struct{}
+
+type Resource struct {
+	RemoteIdentifier string `json:"remoteIdentifier"`
+}
+
+type ValetRequestParams struct {
+	Operation string `json:"operation"`
+	Resources []Resource
+}
+
+type ValetToken struct {
+	Authorization string `json:"authorization"`
+	FileId        string `json:"fileId"`
```

Comment:
Golang's convention:
```suggestion
	FileID        string `json:"fileId"`
```